functions/Invoke-DbaAdvancedRestore.ps1

```diff
@@ -234,13 +234,17 @@ function Invoke-DbaAdvancedRestore {
                     Write-Message -Message "Adding device $file" -Level Debug
                     $Device = New-Object -TypeName Microsoft.SqlServer.Management.Smo.BackupDeviceItem
                     $Device.Name = $file
-                    if ($file -like "http*") {
+                    if ($file -match 'http') {
```

Comment:
and if my database name is I_love_HTTP_appname ..... this breaks because it searches for http throughout all the string ... before was strictly more correct, searching just at begin

Reply:
LOL k ty

Reply:
changed it to StartsWith